src/parsing/parser.rs

```diff
@@ -5129,6 +5140,7 @@ fn parse_parameters_or_arguments<'a, F>(opts: ParseParametersOrArgumentsOptions<
                 multi_line_options: parser_helpers::MultiLineOptions::surround_newlines_indented(),
                 force_possible_newline_at_start: is_parameters,
                 node_sorter: None,
+                prefer_inline_multi_line: !is_parameters,
```

Comment:
Everything looks good! Thanks a lot, @kdy1! I am only not sure about this. I'll have to think about it a little bit and will merge over the next few days. I think there are probably some scenarios this affects that aren't properly tested, but I will add some tests if I think of anything :)

Reply:
Thanks! I'll also try to find such cases.
I think some call expressions with complex arguments can be candidate.